### ISSUE TITLE
Make it easier to find Lazydocker's default config file

### ISSUE THREAD

**Issue by Tset-Noitamotua** [jesseduffield/lazydocker#174]
Each in a while (every few months) when I want to add a new custom command I struggle to remember where to find Lazydocker's config file. I have to go to Github and check the readme to find out.

**Enhancement Suggestion**
Instead of having the default config file in `~/.config/jesseduffield/lazydocker/` it would be easier to remember the location if it was in `~/.config/lazydocker/`

**Describe alternatives you've considered**
A workaround is to define a custom location during install. But that could lead to the problem that I forget my custom location and then even checking readme on Github wouldn't help :)

**Comment by mjarkk** [comment]
You can also go to the status panel and press `e`  
Or link the config file `ln -s ~/.config/jesseduffield/lazydocker/config.yml ~/.config/lazydocker/config.yml`.  

I personally don't have to many problems with the current path because it respects the way go projects work but i also agree that the `jesseduffield` folder inside `.config` is a bit wired.

**Comment by jesseduffield** [comment]
yeah I'm open to suggestions on this one. The reason I used the jesseduffield/lazydocker approach was because it matched the XDG spec (I think) of having creator/application-name so I just mimicked the github path.

**Comment by dawidd6** [comment]
XDG spec is one thing and almost everyone is using `.config/$NAME` directory for config files anyway. Like `.config/fish`, `.config/nvim`, etc. Or even better, dump config files directly in `.config` like `hub` does for example.

So, tbh, I would go for `.config/lazygit` too, as the name is quite unique I think. But we would need to create some migration mechanism, from old dir to new.

**Comment by glvr182** [comment]
The XDG specification does afaik not talk about vendor names. It only mentions `$HOME/.config`.
The library states the following: 
```
When creating XDG application the Vendor and Application names are appended to the end of the path to keep projects unique.
```
So I'd say it is just to keep things unique.
As result I would also go for `.config/lazygit` since this is what the majority of the programs uses aswell
